lib/rmt/mirror/base.rb

```diff
@@ -132,18 +133,22 @@ def need_to_download?(ref)
   end
 
   def cleanup_stale_metadata
-    # A bug introduced in 2.16 writes metadata into its own directory if it exists
-    # resulting in a directory structure like repodata/repodata.
+    logger.debug('Removing stale metadata files...')
```

Comment:
do we really want the logging here? We had it for development. Do you think we should keep it?

Reply:
Maybe. Should the user know that the stale data was removed